_pkgdown.yml

```diff
@@ -21,10 +21,14 @@ reference:
 
 - title: Individual linters
   contents:
-  - all_undesirable_functions
   - ends_with("linter")
 
-- title: Customer linters
+- title: Lists of defaults
```

Comment:
I think we can do better on the name here but nothing coming straight to mind... mind having another go at it? I'll also try with a fresh mind tomorrow.

Reply:
A few options from my side:

- Defaults
- Default options
- Customizable components

Reply:
How about "Common default configurations"?

Reply:
Sounds good. Changed to that.